src/main/scala/GuessWho/Interface.scala

```diff
@@ -0,0 +1,159 @@
+package GuessWho
+
+import scala.io.StdIn.readLine
+
+class Interface {
+  private var _hairQuestions:Map[Int, String] = Map(
```

Comment:
None of these need to be `var`s I think...

Another place to use enums for questions too!